pkg/doc/signature/jsonld/processor_test.go

```diff
@@ -273,7 +273,7 @@ const (
   ]
 }
 `
-	// nolint
+	// nolint.
```

Comment:
This was needed? Also odd that the prior code had an unscoped nolint. Similarly below.